README.md

```diff
@@ -19,3 +19,18 @@ Installation
 ------------
 
 `qutip-cupy` is not yet officially released.
+
+If you want to try out the package you will need to have a CUDA enabled GPU, `QuTiP >5.0.0` and `CuPy`.
+We recommend using a conda environment `Python >= 3.7`.
+To install `CuPy` we recommend the following steps:
+
+- conda install cudatoolkit
```

Comment:
This line is redundant as `cudatoolkit` is a dependency of `cupy` which gets handled in the next line.
```suggestion
```